### PR TITLE
[Merged by Bors] - chore(NumberTheory/Numberfield/*): rename ringEquiv_complex_of_isComplex -> ringEquivComplexOfIsComplex

### PR DESCRIPTION
**Pull request by kbuzzard**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

Suggested in a [reviewer comment](https://github.com/leanprover-community/mathlib4/pull/19646/files#r1864687306) in #19646 

This PR makes the following renames (and deprecates the old defs):

`def extensionEmbedding_of_isReal` -> `def extensionEmbeddingOfIsReal`
`def ringEquiv_complex_of_isComplex` -> `def ringEquivComplexOfIsComplex`
`def isometryEquiv_complex_of_isComplex` -> `def isometryEquivComplexOfIsComplex`
`def ringEquiv_real_of_isReal` -> def `ringEquivRealOfIsReal`
`def isometryEquiv_real_of_isReal` -> def `isometryEquivRealOfIsReal`

### CODE REVIEW

File: Mathlib/NumberTheory/NumberField/Completion.lean [hunk 1]
```diff
@@ -134,14 +134,17 @@ theorem bijective_extensionEmbedding_of_isComplex {v : InfinitePlace K} (hv : Is
...
-def ringEquiv_complex_of_isComplex {v : InfinitePlace K} (hv : IsComplex v) :
+def ringEquivComplexOfIsComplex {v : InfinitePlace K} (hv : IsComplex v) :
     v.completion ≃+* ℂ :=
   RingEquiv.ofBijective _ (bijective_extensionEmbedding_of_isComplex hv)
 
+@[deprecated (since := "2024-12-01")]
+noncomputable alias ringEquiv_complex_of_isComplex := ringEquivComplexOfIsComplex
+
 /-- If the infinite place `v` is complex, then `v.completion` is isometric to `ℂ`. -/
 def isometryEquiv_complex_of_isComplex {v : InfinitePlace K} (hv : IsComplex v) :
```
Comment (Ruben-VandeVelde) [inline Mathlib/NumberTheory/NumberField/Completion.lean:145]: Same here, btw

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge 

Thanks!

**Comment by riccardobrasca** [comment]
I think you can follow Ruben's comment and change `isometryEquiv_complex_of_isComplex` in this PR. Thanks!

bors d+

**Comment by urkud** [comment]
BTW, there are more `*_*` definitions in this file.

File: Mathlib/NumberTheory/NumberField/Completion.lean (hunk 1 above)
Comment (kbuzzard) [reply Mathlib/NumberTheory/NumberField/Completion.lean]: Fixed. Thanks!

**Comment by kbuzzard** [comment]
Yes Yury's right, there are three more!

Summary of this PR currently: 

`def extensionEmbedding_of_isReal` -> `def extensionEmbeddingOfIsReal`
`def ringEquiv_complex_of_isComplex` -> `def ringEquivComplexOfIsComplex`
`def isometryEquiv_complex_of_isComplex` -> def isometryEquivComplexOfIsComplex`
`def ringEquiv_real_of_isReal` -> `def ringEquivRealOfIsReal`
`def isometryEquiv_real_of_isReal` -> `def isometryEquivRealOfIsReal`

Can someone check to see if I've missed any more before I merge?

File: Mathlib/NumberTheory/NumberField/Completion.lean [hunk 2]
```diff
@@ -152,35 +155,47 @@ theorem bijective_extensionEmbedding_of_isComplex {v : InfinitePlace K} (hv : Is
...
     v.Completion ≃ᵢ ℂ where
-  toEquiv := ringEquiv_complex_of_isComplex hv
+  toEquiv := ringEquivComplexOfIsComplex hv
   isometry_toFun := isometry_extensionEmbedding v
 
-/-- If `v` is a real infinite place, then the embedding `v.Completion →+* ℝ` is surjective. -/
+@[deprecated (since := "2024-12-07")]
+noncomputable alias isometryEquiv_complex_of_isComplex := isometryEquivComplexOfIsComplex
+
+/-- If `v` is a real infinite place, then the embedding `v.completion →+* ℝ` is surjective. -/
```
Comment (kbuzzard) [inline Mathlib/NumberTheory/NumberField/Completion.lean:174]:
```suggestion
/-- If `v` is a real infinite place, then the embedding `v.Completion →+* ℝ` is surjective. -/
```

File: Mathlib/NumberTheory/NumberField/Completion.lean [hunk 3]
```diff
@@ -152,35 +155,47 @@ theorem bijective_extensionEmbedding_of_isComplex {v : InfinitePlace K} (hv : Is
...
+def ringEquivRealOfIsReal {v : InfinitePlace K} (hv : IsReal v) : v.Completion ≃+* ℝ :=
   RingEquiv.ofBijective _ (bijective_extensionEmbedding_of_isReal hv)
 
-/-- If the infinite place `v` is real, then `v.Completion` is isometric to `ℝ`. -/
-def isometryEquiv_real_of_isReal {v : InfinitePlace K} (hv : IsReal v) : v.Completion ≃ᵢ ℝ where
-  toEquiv := ringEquiv_real_of_isReal hv
+@[deprecated (since := "2024-12-07")]
+noncomputable alias ringEquiv_real_of_isReal := ringEquivRealOfIsReal
+
+/-- If the infinite place `v` is real, then `v.completion` is isometric to `ℝ`. -/
```
Comment (kbuzzard) [inline Mathlib/NumberTheory/NumberField/Completion.lean:193]:
```suggestion
/-- If the infinite place `v` is real, then `v.Completion` is isometric to `ℝ`. -/
```

**Comment by kbuzzard** [comment]
I think I'm just being over-cautious. I checked the file again after sleeping on it, and I'm pretty sure that it's now OK.

bors r+

**Pull request closed without merging**